mautrix_telegram/db/portal.py

```diff
@@ -54,8 +54,13 @@ def find_private_chats(cls, tg_receiver: TelegramID) -> Iterable['Portal']:
         yield from cls._select_all(cls.c.tg_receiver == tg_receiver, cls.c.peer_type == "user")
 
     @classmethod
-    def get_by_mxid(cls, mxid: RoomID) -> Optional['Portal']:
-        return cls._select_one_or_none(cls.c.mxid == mxid)
+    def count(cls) -> int:
+        rows = cls.db.execute(select([func.count()]))
+        try:
+            count, = next(rows)
+            return count
+        except StopIteration:
+            return 0
```

Comment:
Baby's first Python commits (don't commit blindly):

```suggestion
        return cls.db.query(func.count('*')).scalar()
```

Please test this. I haven't actually run this.

Reply:
No good unfortunately, 

`AttributeError: 'Engine' object has no attribute 'query'`

Reply:
The table to count rows in probably needs to be specified somewhere :thinking: